README.md

```diff
@@ -53,6 +53,19 @@ m.addCourse(965389)   # BE 511 Business Economics II
 m.downloadAllFiles()
 ```
 
+If you want to download all your courses without adding their ref ids 
+manually you can do it this way:
+
+```python
+from IliasDownloaderUniMA import IliasDownloaderUniMA
+
+m = IliasDownloaderUniMA()
+m.setParam('download_path', '/Users/jonathan/Desktop/')
+m.login('jhelgert', 'my_password')
+m.addAllUserCourses()
```

Comment:
```suggestion
m.addAllCurrentSemesterCourses()
```

Reply:
See my comment to your first addAllUserCourses proposal.
TL;DR I'd pick addAllSemesterCourses, the method name is even with addAllSemesterCourses already a bit too long imo, but borderline okayish I'd say.